io-engine/src/lvm/lv_replica.rs

```diff
@@ -0,0 +1,739 @@
+use super::{cli::de, error::Error, vg_pool::VolumeGroup, CmnQueryArgs};
+use crate::{
+    bdev::PtplFileOps,
+    bdev_api::{bdev_create, BdevError},
+    core::{Protocol, PtplProps, Share, ShareProps, UntypedBdev, UpdateProps},
+    lvm::{
+        cli::LvmCmd,
+        property::{Property, PropertyType},
+    },
+};
+
+use std::{
+    ops::{Deref, DerefMut},
+    pin::Pin,
+};
+
+/// Different list options for a logical volume.
+#[derive(Default, Debug)]
+pub(crate) struct QueryArgs {
+    /// Pertaining the volume group parent.
+    vg: super::vg_pool::QueryArgs,
+    /// Pertaining the logical volume itself.
+    lv: CmnQueryArgs,
+    /// Our lv's are "special", as in we set the uuid to the name, and add the
+    /// name as a lv_tag (LVM_TAG.lv.name=name)
+    regular_lv: bool,
```

Comment:
What do we mean by add name as lv_tag? What is the name here? Volume uuid?

Reply:
Also de we expect lv name to be uuidv4 always? If so can it be an issue while loading non mayastor created LVs?

Reply:
As per the comment, we add the replica name as a tag, eg: mayastor.lv.name=replica1
If loading non-mayastor LVs then probably LV will have to be renamed; 
We might be able to do the opposite and create with name, setting up uuid as mayastor.lv.uuid=uuid but then the lv_path would have the name and not the uuid.